src/app/zeko/da_layer/lib/batch.ml

```diff
@@ -0,0 +1,35 @@
+open Core_kernel
+open Mina_base
+open Mina_ledger
+
+[%%versioned
+module Stable = struct
+  module V1 = struct
+    type t =
+      { source_ledger_hash : Ledger_hash.Stable.V1.t
+      ; target_ledger_hash : Ledger_hash.Stable.V1.t
```

Comment:
Why include target here

Reply:
Hmm yeah this is redundant. I've added it for convenience because sequencer needs to know target ledger hash beforehand to know under which key store the deferred signatures. This however doesn't make much sense.

I'll change the api so that `enqueue_distribute_batch` takes the `target_ledger_hash`.